### ISSUE TITLE
Make an init script for windows?

### ISSUE THREAD

**Issue by GaelTouquet** [GaelTouquet/Geometry#4]
try and find something equivalent to init.sh for windows, so that the pythonpath is set?...

**Comment by GaelTouquet** [comment]
Maybe use venv or conda envs ? 

if conda envs, maybe use environment.yml? Although there is no direct VSCode integration (https://code.visualstudio.com/docs/python/environments)

**Comment by GaelTouquet** [comment]
stipulate the pythonpath to Geometry in a .env file?

**Comment by GaelTouquet** [comment]
Should add scipy and numpy to requirements.txt?

**Comment by NoZiL** [comment]
yep yep! I commented something like that on the PR. To make it short, if you install/use a dep, you should list it in requirements.txt
